wasi-sdk.cmake

```diff
@@ -3,6 +3,10 @@
 # This is arbitrary, AFAIK, for now.
 cmake_minimum_required(VERSION 3.4.0)
 
+# Until Platform/WASI.cmake is upstream we need to inject the path to it
+# into CMAKE_MODULE_PATH.
+list(APPEND CMAKE_MODULE_PATH "${WASI_SDK_PREFIX}/share/cmake")
```

Comment:
Can we use a path relative to the toolchain file itself?  or at least default to a relative path so its not required to always pass the extra WASI_SDK_PREFIX argument?

Reply:
Probably not: When building WASI-SDK itself `WASI_SDK_PREFIX` is not in the correct relative spot compared to this file yet. If I'm interpreting the Makefile logic correctly. But I can give it a try.

And WASI_SDK_PREFIX is already needed further down in the file for finding the compilers as well (so I'm not introducing a new thing). Do we want to make those use relative paths to the cmake file too?

Reply:
Turns out I was wrong, we can do this. I have not changed the use further down for locating the compilers though